site/docs/tools/deltalake_client.md

```diff
@@ -0,0 +1,200 @@
+# Delta Lake Client
+
+!!! note
+    You can follow along interactively in a Jupyter notebook by following the instructions
+    [here](https://github.com/projectnessie/nessie/tree/main/python/demo).
+
+Delta Lake support in Nessie requires some minor modifications to the core Delta libraries. This patch is still ongoing,
```

Comment:
Just to clarify: this is just moving the DL-part from the `spark.md` file, right?

Reply:
Yes, this is just moving DL docs out of `spark.md` into `datalake_client.md`.